client/state/billing-transactions/test/reducer.js

```diff
@@ -235,4 +244,41 @@ describe( 'reducer', () => {
 			expect( state ).to.eql( {} );
 		} );
 	} );
+
+	describe( '#individualTransactions', () => {
+		const receiptId = 345;
+
+		const currentState = {};
+
+		test( 'BILLING_RECEIPT_REQUEST', () => {
+			const state = individualTransactions( currentState, {
+				type: BILLING_RECEIPT_REQUEST,
+				receiptId,
+			} );
+
+			expect( state.requesting[ receiptId ] ).to.eql( true );
+		} );
+
+		test( 'BILLING_RECEIPT_REQUEST_SUCCESS', () => {
+			const state = individualTransactions( currentState, {
+				type: BILLING_RECEIPT_REQUEST_SUCCESS,
+				receiptId,
+				receipt: { data: 'test receipt' },
+			} );
+
+			expect( state.requesting[ receiptId ] ).to.eql( false );
+			expect( state.receipts[ receiptId ] ).to.eql( { data: 'test receipt' } );
+		} );
+
+		test( 'BILLING_RECEIPT_REQUEST_FAILURE', () => {
+			const state = individualTransactions( currentState, {
+				type: BILLING_RECEIPT_REQUEST_FAILURE,
+				receiptId,
+				error: true,
+			} );
+
+			expect( state.requesting[ receiptId ] ).to.eql( false );
```

Comment:
Here you can see a good example of why it would be better if we keep the `requesting` and `items` for the individual transactions in separate reducers.